libs/shared/db/mysql/account/src/lib/cart.ts

```diff
@@ -12,20 +12,21 @@ export class Cart extends BaseModel {
   protected $uuidFields = ['id', 'uid'];
 
   // table fields
-  id!: string;
+  readonly id!: string;
   uid?: string;
   state!: CartState;
   errorReasonId?: string;
   offeringConfigId!: string;
   interval!: string;
   experiment?: string;
   taxAddress?: TaxAddress;
-  createdAt!: number;
-  updatedAt!: number;
+  readonly createdAt!: number;
+  readonly updatedAt!: number;
```

Comment:
Just a note, this is `objection` ORM stuff, so it'll be removed entirely in the future.